helm-charts/hbase-chart/templates/_poddistributionbudget.tpl

```diff
@@ -0,0 +1,13 @@
+{{- define "com.flipkart.poddistributionbudgets" }}
+---
+{{- include "createPDB" (dict "deployment" .Values.deployments.zookeeper "pdbConfig" .Values.deployments.zookeeper.pdbConfig "component" "zk" "namespace" .Values.namespace "instance" .Values.service.name) }}
```

Comment:
why not reususe instead of doing it one per component? can you read through the code of how it is done for other parts